R/ggcuminc.R

```diff
@@ -55,6 +55,14 @@ ggcuminc <- function(x, outcome = NULL,
     )
   }
 
+  # Generate an error if the input is a `survfit.coxphms` regression model: ----
+  if (inherits(x, "survfitcoxms")) {
+    cli_abort(
+      "Argument {.arg x} does not support {.cls survfit.coxphms} object.
+  Please fit a multi-state model with another project, such as {.fn coxph}."
```

Comment:
This is great, thank you for adding. Let's keep the message simple with `"Argument {.arg x} does not support {.cls survfit.coxphms} object."`.

Can you also add a unit test to test appropriate error messaging with `expect_error()`?

Reply:
Thanks @ddsjoberg, I'll work with the unit test later (on Issue #222 PR)